src/src/PluginStructs/P104_data_struct.cpp

```diff
@@ -221,7 +221,7 @@ void P104_data_struct::loadSettings() {
           zones[zoneIndex].size = tmp_int;
         }
 
-        zones[zoneIndex].text = parseStringKeepCase(tmp, 1 + P104_OFFSET_TEXT, P104_FIELD_SEP);
+        zones[zoneIndex].text = parseStringKeepCaseNoTrim(tmp, 1 + P104_OFFSET_TEXT, P104_FIELD_SEP);
```

Comment:
Ditto

Reply:
It is stored with any leading/trailing spaces, so have to load it like that also.